docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -10,27 +10,42 @@ Follow this guide to deploy and run the service.
          ```bash
          cd edgex-compose/compose-builder/
          ```
+      2. Run Edgex with the ONVIF microservice in secure or non-secure mode.
 
-      2. Run EdgeX with the microservice in non-secure mode:
+        ##### Non-secure mode
 
-         ```bash
-         make run no-secty ds-onvif-camera
-         ```
-      
-      3. Run EdgeX with the microservice in secure mode:
+        ```bash
+        make run no-secty ds-onvif-camera
+        ```
+    
+        ##### Secure mode 
+
+        !!! note
+            Recommended for secure and production level deployments. 
 
          ```bash
          make run ds-onvif-camera
          ```
+        !!! note
+            Need to wait for sometime for the services to be fully up before executing the next set of commands.
+            Make a note of the Consul ACL token and the JWT token generated which are needed to map credentials and execute apis.
```

Comment:
probably should instruct the users to store it in a secure manner (instead of "make a note"), as these keys are very critical to the security of the EdgeX services. In addition, warn them not to store them in cleartext on the machine.